### ISSUE TITLE
Documentation needs work and there are dead links

### ISSUE THREAD

**Issue by tetreault** [jeromeetienne/AR.js#223]
I really like this lib and have been following it for a while and plan on using this. Documentation is really haphazardly scattered thru out the repo and theres dead links galore ([this](https://jeromeetienne.github.io/AR.js/three.js/examples/magic-book/index.html) from the examples page for example). 

What can we do to fix this. How can I help you out here. The average dev might not be enticed to pick up such a great library like this because aside from the 10-line-markup example documentation is all over the place.

**Comment by mcdtech** [comment]
Would like to know if there is any update on documentation.  Seems like a great library but really hard to follow.

**Comment by ZoltanVeres** [comment]
We are working on documentation and restructuring the demos.

**Comment by vincentorback** [comment]
If anyone has bookmarked where the documentation is scattered through out the repo and have links that would be much appreciated :)

**Comment by nicolocarpignoli** [comment]
duplicate of https://github.com/jeromeetienne/AR.js/issues/362
